src/QsCompiler/SyntaxProcessor/ExpressionVerification.fs

```diff
@@ -246,11 +246,17 @@ let private VerifyConcatenation parent addError (lhsType : ResolvedType, lhsRang
 /// If a common base type exists, verifies that this base type supports equality comparison, 
 /// adding the corresponding error otherwise.
 /// If one of the given types is a missing type, also adds the corresponding ExpressionOfUnknownType error(s).
-let private VerifyEqualityComparison parent addError (lhsType : ResolvedType, lhsRange) (rhsType : ResolvedType, rhsRange) =
-    // NOTE: this may not be the behavior that we want (right now it does not matter, since we don't support equality comparison for any derived type)
-    let baseType = CommonBaseType addError (ErrorCode.ArgumentMismatchInBinaryOp, [lhsType |> toString; rhsType |> toString]) parent (lhsType, lhsRange) (rhsType, rhsRange)
-    let expected (t : ResolvedType) = t.supportsEqualityComparison
-    VerifyIsOneOf expected (ErrorCode.InvalidTypeInEqualityComparison, [baseType |> toString]) addError (baseType, rhsRange) |> ignore
+let private VerifyEqualityComparison context addError (lhsType, lhsRange) (rhsType, rhsRange) =
+    // NOTE: this may not be the behavior that we want (right now it does not matter, since we don't support equality
```

Comment:
I am not sure what you mean by that actually? What behavior do you think should be different if we were to support equality comparison for operations?

Reply:
I believe that you wrote this comment originally. :) I only reformatted the lines.

Reply:
Hu, that comment has just enough information that something may be good thinking about without actually biasing the reader regarding what should be considered... 
While I need to write better comments, let's keep it - I've found in the past that often when I ignored a remark about something needing to be considered, I'd better not. Past me when writing the code tends to have better insights into it than current me.